CHANGELOG.md

```diff
@@ -41,7 +41,8 @@ The types of changes are:
 * Integrate rate limiter with Saas Connectors. [#1433](https://github.com/ethyca/fides/pull/1433)
 * Config Wizard: Added a column selector to the scan results page of the config wizard [#1590](https://github.com/ethyca/fides/pull/1590)
 * Config Wizard: Flow for runtime scanner option [#1640](https://github.com/ethyca/fides/pull/1640)
-* Access support for Twilio Conversations API [#1520](https://github.com/ethyca/fides/pull/1520)
+* Erasure support for Twilio Conversations API [#1673](https://github.com/ethyca/fides/pull/1673)
```

Comment:
This is my fault for waiting so long to review but can you move this line to the `Unreleased` section and bring back the access support entry?